changelog/fix_fix_false_negatives_for_ rails_redundant_active_record_all_method.md

```diff
@@ -0,0 +1 @@
+* [#1382](https://github.com/rubocop/rubocop-rails/pull/1382): Fix false negatives for `Rails/RedundantActiveRecordAllMethod`. ([@masato-bkn][])
```

Comment:
The change log could use a bit more information. For example, something like the following:

```suggestion
* [#1382](https://github.com/rubocop/rubocop-rails/pull/1382): Fix false negatives for `Rails/RedundantActiveRecordAllMethod` when using `all` method in block. ([@masato-bkn][])
```